modules/installation-installer-provisioned-vsphere-config-yaml.adoc

```diff
@@ -23,67 +23,74 @@ endif::[]
 [id="installation-installer-provisioned-vsphere-config-yaml_{context}"]
 = Sample install-config.yaml file for an installer-provisioned VMware vSphere cluster
 
-You can customize the install-config.yaml file to specify more details about
+You can customize the `install-config.yaml` file to specify more details about
 your {product-title} cluster's platform or modify the values of the required
 parameters.
 
+[NOTE]
```

Comment:
IMO this note is not necessary. Based on our recent conversations, I understand that you do not want people to miss anything. However, this sample does contain the `failureDomains` and `topology` stanzas. I believe a more elegant way of handling this is to annotate `failureDomains` and to cross referencing "Configuring regions and zones for a VMware vCenter" from there. Based on the placement of `failureDomains`, this would only require you to bump a handful of annotations. My suggestion is to let the sample stand on its own and let the annotations do the talking.

Reply:
Hi @mjpytlak . An update to the machine management Sample YAML files is not required, so I updated the two sample YAML installation files based on your feedback.